stdlib/tkinter/__init__.pyi

```diff
@@ -162,7 +162,7 @@ EXCEPTION = _tkinter.EXCEPTION
 # Don't trust tkinter's docstrings, because they have been created by copy/pasting from
 # Tk's manual pages more than 10 years ago. Use the latest manual pages instead:
 #
-#    $ sudo apt install tk-doc tcl-doc
+#    $ sudo apt-get install tk-doc tcl-doc
```

Comment:
`apt-get` is recommended for use in scripts while `apt` is still the recommended tool for interactive use. All other changes look good to me except this one which I think is better left unchanged as it describes commands to be executed in the terminal.